### ISSUE TITLE
How to downgrade a realm file format version?

### ISSUE THREAD

**Issue by caeduk** [realm/realm-kotlin#1845]
### Problem

Is it possible to revert to a previous realm version format by using the realm-kotlin SDK?
I.e downgrading from v24 to v23

### How important is this improvement for you?

Would be a major improvement

### Feature would mainly be used with

Local Database only

**Comment by BlueCobold** [comment]
I don't think this is possible unless you do it yourself by exporting all data on a higher version and creating a new lower version realm. Which is not possible within the same application, I think.
